CMakeLists.txt

```diff
@@ -200,6 +200,7 @@ FIND_PACKAGE(GDAL)
 FIND_PACKAGE(Expat REQUIRED)
 FIND_PACKAGE(Spatialindex REQUIRED)
 FIND_PACKAGE(Qwt REQUIRED)
+FIND_PACKAGE(QJSON)
```

Comment:
The WMS provider uses [QScriptEngine](http://doc.qt.io/qt-4.8/qscriptengine.html) to parse feature infos in JSON (see [qgswmsprovider.cpp:2621](https://github.com/qgis/QGIS/blob/master/src/providers/wms/qgswmsprovider.cpp#L2621))